test/azure/terraform_azure_keyvault_example_test.go

```diff
@@ -0,0 +1,61 @@
+// +build azure
+
+// NOTE: We use build tags to differentiate azure testing because we currently do not have azure access setup for
+// CircleCI.
+
+package test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/azure"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTerraformAzureKeyVaultExample(t *testing.T) {
+	t.Parallel()
+
+	uniquePostfix := random.UniqueId()
+
+	expectedSecretName := fmt.Sprintf("secret-%s", random.UniqueId())
```

Comment:
shouldn't these be fmt.Sprintf("secret-%s", uniquePostfix))

Reply:
refactored